build/cluster-operator-ansible/playbooks/cluster-api-prep/files/cluster-api-template.yaml

```diff
@@ -33,73 +33,73 @@ objects:
 - apiVersion: apiregistration.k8s.io/v1beta1
   kind: APIService
   metadata:
-    name: v1alpha1.cluster.k8s.io
     labels:
       api: clusterapi
+    name: v1alpha1.cluster.k8s.io
   spec:
-    version: v1alpha1
+    caBundle: ${SERVING_CA}
     group: cluster.k8s.io
     groupPriorityMinimum: 2000
-    priority: 200
     service:
       name: clusterapi
       namespace: ${CLUSTER_API_NAMESPACE}
+    version: v1alpha1
     versionPriority: 10
-    caBundle: ${SERVING_CA}
 
 - apiVersion: v1
   kind: Service
   metadata:
-    name: clusterapi
-    namespace: ${CLUSTER_API_NAMESPACE}
     labels:
       app: cluster-api-apiserver
+    name: clusterapi
+    namespace: ${CLUSTER_API_NAMESPACE}
   spec:
     ports:
     - port: 443
       protocol: TCP
       targetPort: 6443
     selector:
       app: cluster-api-apiserver
+    sessionAffinity: None
+    type: ClusterIP
 
-- apiVersion: apps/v1beta1
+- apiVersion: extensions/v1beta1
   kind: Deployment
   metadata:
-    name: cluster-api-apiserver
-    namespace: ${CLUSTER_API_NAMESPACE}
+    generation: 1
```

Comment:
Does setting this have something to do with getting oc_process to not recreate? 

This makes me a little nervous, doesn't seem like something we should be setting and I'm also wondering if changing the yaml would get applied if you didn't bump the generation?

kubectl apply has the logic we want and sadly it's not server side yet, so anything that's doing it's own thing like oc process makes me a little nervous as there's a lot of complexity in it. Could we do what we do for our dev template for now? https://github.com/openshift/cluster-operator/blob/master/contrib/ansible/deploy-devel.yaml#L58

In future this will hopefully be k8s_raw + server side apply logic.